OrbitGl/ModulesDataView.cpp

```diff
@@ -136,23 +154,33 @@ std::vector<std::string> ModulesDataView::GetContextMenu(
 void ModulesDataView::OnContextMenu(const std::string& a_Action,
                                     int a_MenuIndex,
                                     const std::vector<int>& a_ItemIndices) {
-  if (a_Action == MENU_ACTION_MODULES_LOAD) {
+  if (a_Action == MENU_ACTION_MODULES_LOAD ||
```

Comment:
Can we have two separate ifs for the two actions? A few lines of code might be duplicated but it's more readable than having the logics for the two actions interleaved.